src/components/childComponents/table/DislocationResultsTable.jsx

```diff
@@ -126,17 +134,16 @@ export function DislocationResultsTable(props) {
 							<TableCell className={classes.tableColumnHeadRed}> Dislocated </TableCell>
 							<TableCell className={classes.tableDataCellDislocated}>
 								{
-									"households" in resultsJson.dislocated ?
-										resultsJson.dislocated["households"].toLocaleString("en-US")
-										:
-										resultsJson.dislocated["population"].toLocaleString("en-US")
+									"households" in resultsJson["dislocated"] ?
+										resultsJson["dislocated"]["households"].toLocaleString("en-US") :
+										resultsJson["dislocated"]["population"].toLocaleString("en-US")
 								}
 							</TableCell>
 							<TableCell className={classes.tableDataCellDislocated}>
 								{
-									"%_of_households" in resultsJson.dislocated ?
-										round(resultsJson.dislocated["%_of_households"], 1):
-										round(resultsJson.dislocated["%_of_population"], 1)
+									"%_of_households" in resultsJson["dislocated"] ?
```

Comment:
not for this PR, but I think it is not best practice to begin the json keys with non-alphabetic characters, because at some point they might get casted to javascript objects that can't handle them using the "dot" notation. There could be other implications too

Reply:
Yeah I was thinking of the same but it got too close to the demo and this requires a pyincore change. I will try to see if we can address that in this https://github.com/IN-CORE/pyincore/pull/44@ncsa-mo